packages/test-runner-intercept/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@web/test-runner-intercept",
+  "version": "0.1.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Plugin for intercepting modules in @web/test-runner",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/modernweb-dev/web.git",
+    "directory": "packages/test-runner-intercept"
+  },
+  "author": "modern-web",
+  "homepage": "https://github.com/modernweb-dev/web/tree/master/packages/test-runner-intercept",
+  "main": "browser/index.js",
+  "exports": {
+    ".": {
+      "types": "./browser/index.d.ts",
+      "default": "./browser/index.mjs"
+    },
+    "./plugin": {
+      "types": "./plugin.d.ts",
+      "import": "./plugin.mjs",
+      "require": "./dist/interceptModulePlugin.js"
```

Comment:
We should just go ESM-only, we're already looking into moving everything to ESM anyway

Reply:
I think doing this before the PR to ESM might be more tricky. Because there are scripts that need to be adapted for it.

Reply:
How so? We did already move some of the projects to ESM, like dev-server-storybook. Which scripts would need to be adjusted?

Reply:
This is now updated. :-) Can you check and resolve if ok?